packages/lpa-questionnaire-web-app/package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@ministryofjustice/frontend": "0.2.0",
     "@pins/common": "file:../common",
+    "@planning-inspectorate/pins-clamav": "^1.0.8",
```

Comment:
As above - Incorrect package name and version? Should be pins-clamav-rest-client?

Reply:
Done